src/plugins/visualizations/common/convert_to_lens/constants.ts

```diff
@@ -68,7 +71,7 @@ export const LayerTypes = {
   ANNOTATIONS: 'annotations',
 } as const;
 
-export const XYCurveTypes = {
+export const XYCurveTypes: Record<string, $Keys<typeof CurveType>> = {
```

Comment:
I was always wondering about the reason to duplicate this enum, do you have ideas @nickofthyme ?

If this is done to detach the implementation from elastic-charts probably is better to call things differently and have a mapping between this and elastic-charts somewhere else, but other than that I don't understand that fully